src/main/kotlin/org/genspectrum/ingest/workflow/SC2GisaidWorkflow.kt

```diff
@@ -8,18 +8,7 @@ import org.genspectrum.ingest.AlignedGenome
 import org.genspectrum.ingest.file.Compression
 import org.genspectrum.ingest.file.File
 import org.genspectrum.ingest.file.FileType
-import org.genspectrum.ingest.proc.HashEntry
-import org.genspectrum.ingest.proc.compareHashes
-import org.genspectrum.ingest.proc.concatFiles
-import org.genspectrum.ingest.proc.extractAddedOrChanged
-import org.genspectrum.ingest.proc.extractUnchanged
-import org.genspectrum.ingest.proc.fastaToNdjson
-import org.genspectrum.ingest.proc.joinSC2GisaidData
-import org.genspectrum.ingest.proc.renameFile
-import org.genspectrum.ingest.proc.sortNdjson
-import org.genspectrum.ingest.proc.transformSC2GisaidBasics
-import org.genspectrum.ingest.proc.tsvToNdjson
-import org.genspectrum.ingest.proc.unalignedNucleotideSequencesToFasta
+import org.genspectrum.ingest.proc.*
```

Comment:
It's usually better to avoid wildcard imports (in the worst case you end up using a different class of the same name without noticing).

Reply:
Yeah, I need to configure IntelliJ properly.. I just fixed it.